src/styles/block/nav.scss

```diff
@@ -0,0 +1,27 @@
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+
+    margin: 0;
+    padding: 0;
+  }
+
+  &__link {
+    text-transform: uppercase;
+    text-decoration: none;
+    transition-duration: $transition-speed;
```

Comment:
Make sure to use a variable for transition properties for consistency and maintainability, unless $transition-speed is already a variable that includes the property name as well. If it's just a duration value, consider renaming it to indicate that, or include the property name as well in the value.